editor/import/resource_importer_layered_texture.cpp

```diff
@@ -123,6 +123,9 @@ bool ResourceImporterLayeredTexture::get_option_visibility(const String &p_path,
 	if (p_option == "compress/lossy_quality" && p_options.has("compress/mode")) {
 		return int(p_options["compress/mode"]) == COMPRESS_LOSSY;
 	}
```

Comment:
Didn't you plan to remove lossy quality?

Reply:
Lossy quality is remvoed for VRAM compression, but WebP still uses it.